src/headroom.vue

```diff
@@ -85,7 +85,7 @@ export default {
         'left': '0',
         'right': '0',
         'z-index': this.isInTop ? this.zIndex : 1,
-        'transform': `translateY(${this.translate})`,
+        'transform': `translate3d(0, ${this.translate}, 0)`,
```

Comment:
Yeah, translate3d has better performance. I think we should to detect the translate3d support. For the damned IE9, vue-headroom will still use 2d translate :)